android/README.md

```diff
@@ -88,6 +88,14 @@ adb shell setprop debug.firebase.analytics.app .none.
     * If multiple devices are connected you may need `$ANDROID_HOME/platform-tools/adb.exe install -r -s SERIAL path/to/apk.apk`. Replace `SERIAL` with
        the device serial number listed in step 2.
 
+### Compiling the app's offline help
+Extra prerequisite:
+* `wget` (`brew install wget`)
```

Comment:
Maybe just leave this as a line to install wget. Depending on platform, they may not have brew.